#std_msgs/Header header
std_msgs/String name
std_msgs/Float32 row
std_msgs/Float32 col
std_msgs/Float32 width
std_msgs/Float32 height
sensor_msgs/CompressedImage FaceImg

